DDG4/python/DDG4.py

```diff
@@ -264,16 +272,24 @@ def _get(self, name):
   #  return getattr(self,name)
   #traceback.print_stack()
   msg = 'Geant4Action::GetProperty [Unhandled]: Cannot access property '+a.name()+'.'+name
-  raise exceptions.KeyError(msg)
+  if sys.version_info[0] < 3:
+    import exceptions
```

Comment:
This check and import is not necessary for python2:
https://docs.python.org/2/library/exceptions.html
>>> This module never needs to be imported explicitly: the exceptions are provided in the built-in namespace as well as the exceptions module.
Just dropping `import exceptions` and `exceptions.` should be fine.